### PR TITLE
feat: create custom base component input text

### PR DESCRIPTION
**Pull request by yoslie**
## Changes
- create base component input text
- add asset icon
- add setting json for eslint auto fix
- add nuxt svg loader library

Preview: 

https://user-images.githubusercontent.com/65396086/195474084-55afd926-1e0f-4eaa-95ee-ca683c470e5d.mov

** not available in deploy preview (not called yet)
** svg still have issue bug to change color (will be updated / fix later)

## Evidence
title: Create custom base component input text
project: Jabar Super Apps
participants: @yoslie @Ibwedagama @agunghide @marsellavaleria19

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll": false,
+    "source.fixAll.eslint": true
+  }
+}
```
Comment (Ibwedagama) [inline .vscode/settings.json:6]: ask:

Hello @yoslie, is this ESLint format on save works on your project?

**Review by Ibwedagama — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: components/Base/InputText/index.vue [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+      }
+    }
+  },
+  methods: {
+    /**
+     * handle method when input text on input
+     */
+    onInput (e) {
+      if (e.target.value.length) {
+        this.isTyped = true
```
Comment (marsellavaleria19) [inline components/Base/InputText/index.vue:208]: Ask:  Hello @yoslie, I want to ask what is function of state `isTyped` in `on input` ??

**Review by marsellavaleria19 — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (yoslie) [reply .vscode/settings.json:6]: yes it works thanks for the help ❤️

File: components/Base/InputText/index.vue (hunk 2 above)
Comment (yoslie) [reply components/Base/InputText/index.vue:208]: It is used to define that input has character / length, my expectation is to change icon eye when `isTyped` is `true`

File: .vscode/settings.json (hunk 1 above)
Comment (Ibwedagama) [reply .vscode/settings.json:6]: nice 👍🏼

**Pull request merged**